src/utils/parse-response.ts

```diff
@@ -0,0 +1,63 @@
+import { ExecResponse } from "../types";
+import execAsync from "./exec-async";
+
+const getRequestId = (printResponse: ExecResponse) => {
+  const res = printResponse.stdout;
+  if (res) {
+    // expects the stdout response from lp: 'request id is my-Dummy-Printer-4 (1 file(s))'
```

Comment:
could you please delete the comments? all things that are expected can be tested with unit tests